flink-table/flink-table-planner/src/test/java/org/apache/flink/connector/source/ValuesSource.java

```diff
@@ -52,7 +60,9 @@
  * must be 1. RowData is not serializable and the parallelism of table source may not be 1, so we
```

Comment:
I am struggling to understand this sentence. What must be 1? when we say "RowData is not serializable and the parallelism of table source may not be 1", why is that? A because would be really helpful for the reader.

Reply:
As it is in the scope of this PR, I would prefer to resolve it in a separate doc improvement PR.